python/ingestion/setup.py

```diff
@@ -5,5 +5,5 @@
     package_dir={'ingestion': ''},
     packages=['ingestion'],
     include_package_data=True,
-    package_data={'ingestion': ['merge_data/state_level_fips.csv', 'merge_data/county_level_fips.csv']},
+    package_data={'ingestion': ['merge_data/**/*.csv']},
 )
```

Comment:
Just curious. What does this do?

Reply:
the `package_data` declaration is to let python known which non-python files need to be installed as part of the program, so like in this case the csv files (otherwise the .venv locally won't have access to those files)

but it's not working yet, I tried to use the **/* pattern to copy ALL csvs regardless of subdirectory but it's erroring so im trying some other ways